sdk/spring/azure-spring-data-cosmos/pom.xml

```diff
@@ -7,16 +7,16 @@
     <groupId>com.azure</groupId>
     <artifactId>azure-client-sdk-parent</artifactId>
     <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
-    <relativePath>../../parents/azure-client-sdk-parent</relativePath>
+    <relativePath>../../parents/azure-client-sdk-parent/pom.xml</relativePath>
   </parent>
 
   <groupId>com.azure</groupId>
   <artifactId>azure-spring-data-cosmos</artifactId>
-  <version>3.27.0-beta.1</version> <!-- {x-version-update;com.azure:azure-spring-data-cosmos;current} -->
+  <version>6.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-spring-data-cosmos;current} -->
```

Comment:
Not clear why this would be marked as breaking change (major version upgrade)? We probably need to discuss in a meeting - anytime we introduce a new major version we need to have a plan to deprecate/retire the previous one - and understand that we duplicate support cost.